notebooks/7-rotation-speeds/interm_rotation_rate.ipynb

```diff
@@ -0,0 +1,2926 @@
+{
```

Comment:
<p>Since this is a file that's created in the other notebook, it should also be included in this directory (so someone could, theoretically, run this without having already run the other one.)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/project-tikebook/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='39'/>